.github/workflows/test_dbt_runner.yml

```diff
@@ -9,6 +9,9 @@ on:
   workflow_dispatch:
 
 env:
+
+  DLT_SECRETS_TOML: ${{ secrets.DLT_SECRETS_TOML }}
+
   # all credentials must be present to be passed to dbt runner
   DESTINATION__POSTGRES__CREDENTIALS: postgresql://loader@dlttests.cwz0jfxu0m7m.eu-central-1.rds.amazonaws.com:5432/dlt_data
   DESTINATION__REDSHIFT__CREDENTIALS: postgresql://loader@3.73.90.3:5439/dlt_ci
```

Comment:
why you kept these creds for postgres and redshift?

Reply:
> # all credentials must be present to be passed to dbt runner

got it

Reply:
yeah, these have to be env vars, maybe there is a way we can load the toml file into the env for this, not sure.

Reply:
@AstrakhantsevaAA actually i misinterpreted the test, I could remove this after all